bitnami/external-dns/templates/configmap.yaml

```diff
@@ -1,10 +1,16 @@
-{{- if and (eq .Values.provider "designate") .Values.designate.customCA.enabled }}
+{{- if and (eq .Values.provider "designate") .Values.designate.customCA.enabled | or (and (eq .Values.provider "rfc2136") .Values.rfc2136.rfc3645Enabled) }}
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ template "external-dns.fullname" . }}
   labels: {{ include "external-dns.labels" . | nindent 4 }}
 data:
+{{- if .Values.designate.customCA.enabled }}
   {{ .Values.designate.customCA.filename }}: |
 {{ .Values.designate.customCA.content | indent 4 }}
 {{- end }}
+{{- if .Values.rfc2136.rfc3645Enabled }}
+  krb5.conf: |
+{{ required "rfc2136.kerberosConfig must be set if rfc2136.3645Enabled is `true`" .Values.rfc2136.kerberosConfig | indent 4 }}
```

Comment:
We usually perform this kind of validations in the `_helpers.tpl` file.

Reply:
Done in latest commits

Reply:
This doesn't show as done. It should go to the `_helpers.tpl` file, just as the one you did for the Kerberos realm.